TeslaLib/TeslaClient.cs

```diff
@@ -185,35 +182,7 @@ public async Task LoginUsingTokenStoreAsync(string password, string mfaCode = nu
 
                     token = null;
                 }
-            }
-
-            if (token == null)

```

Comment:
Should we leave this block of code here, but only run it if the password isn't null or empty?